qa/automation/CareerPortal.Tests.Scenarios/Savostin/buttons.feature

```diff
@@ -0,0 +1,52 @@
+Feature: 1.0.0 buttons lead to correct pages
+Background: 
+	Given "Career Quantori" page is opened
+
+Scenario Outline: 1.0.0_01 All buttons of header lead to correct pages
+	When user clicks on <button>
+	Then <page> is opened
```

Comment:
After "Then" it is recommended to use "should do/be done" construction, so I'd rephrase it: "Then ... should be opened"

Reply:
done)